### PR TITLE
fix #61062 sosyalhalisaha.com

### LINKED ISSUE #61062: sosyalhalisaha.com
### Issue URL (Social Widget)
 [https://sosyalhalisaha.com/video-detay/1691306](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsosyalhalisaha.com%2Fvideo-detay%2F1691306)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61062

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -75,6 +75,7 @@ biomedya.com##.share-url
 pravmir.ru##.hh-share-block
 elhow.ru##.art-social-wrapper-ahchor
 elhow.ru##.social-banner
+sosyalhalisaha.com##p.social
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:78]: tag is not necessary here

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:78]: Please add the domain to existing rule
`##.social:not(body)`

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -118,6 +118,7 @@ turkce-brosurler.com##html > body [class^="container-banner-"]
 kriptoarena.com###sidebarbanner
 ||kriptoarena.com/banner/ced.jpg
 ||omeglatv.com/728reklam.gif
+sosyalhalisaha.com#?#.video-list > ul > li:has(> div[id*="ad"])
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:121]: better change it to `#?#.video-list > ul > li:has(> div[id^="div-gpt-ad-"])`

**Comment by Sergey-Lyapin** [comment]
leftover
Screenshot 2020-08-09 at 17 21 37

File: TurkishFilter/sections/specific.txt [hunk 3]
```diff
@@ -118,6 +118,8 @@ turkce-brosurler.com##html > body [class^="container-banner-"]
 kriptoarena.com###sidebarbanner
 ||kriptoarena.com/banner/ced.jpg
 ||omeglatv.com/728reklam.gif
+sosyalhalisaha.com#?#.video-list > ul > li:has(> div[id^="div-gpt-ad-"])
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:121]: to Extended CSS

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**